locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2020-36518   
**Check Name:** jackson-databind: denial of service via a large depth of nested objects   
**Severity:** HIGH   
**Fixed Version:** 2.12.6.1, 2.13.2.1   
**Description:** jackson-databind before 2.13.0 allows a Java StackOverflow exception and denial of service via a large depth of nested objects.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2020-36518